app/dashboard/models.py

```diff
@@ -1127,22 +1127,100 @@ def get_quarterly_stats(self):
             dict : containing the following information
             'user_total_earned_eth': Total earnings of user in ETH.
             'user_total_earned_usd': Total earnings of user in USD.
-            'user_fulfilled_bounties_count': Total bounties fulfilled by user.
+            'user_total_funded_usd': Total value of bounties funded by the user on bounties in done status in USD
+            'user_total_funded_hours': Total hours input by the developers on the fulfillment of bounties created by the user in USD
+            'user_fulfilled_bounties_count': Total bounties fulfilled by user
+            'user_fufilled_bounties': bool, if the user fulfilled bounties
+            'user_funded_bounties_count': Total bounties funded by the user
+            'user_funded_bounties': bool, if the user funded bounties in the last quarter
+            'user_funded_bounty_developers': Unique set of users that fulfilled bounties funded by the user
+            'user_avg_hours_per_funded_bounty': Average hours input by developer on fulfillment per bounty
+            'user_avg_hourly_rate_per_funded_bounty': Average hourly rate in dollars per bounty funded by user
             'user_avg_eth_earned_per_bounty': Average earning in ETH earned by user per bounty
             'user_avg_usd_earned_per_bounty': Average earning in USD earned by user per bounty
             'user_num_completed_bounties': Total no. of bounties completed.
+            'user_num_funded_fulfilled_bounties': Total bounites that were funded by the user and fulfilled
             'user_bounty_completion_percentage': Percentage of bounties successfully completed by the user
+            'user_funded_fulfilled_percentage': Percentage of bounties funded by the user that were fulfilled
             'user_active_in_last_quarter': bool, if the user was active in last quarter
             'user_no_of_languages': No of languages user used while working on bounties.
             'user_languages': Languages that were used in bounties that were worked on.
+            'relevant_bounties': a list of Bounty(s) that would match the skillset input by the user into the Match tab of their settings
         """
         user_active_in_last_quarter = False
+        user_fulfilled_bounties = False
+        user_funded_bounties = False
         last_quarter = datetime.now() - timedelta(days=90)
         bounties = self.bounties.filter(modified_on__gte=last_quarter)
         fulfilled_bounties = [
             bounty for bounty in bounties if bounty.is_hunter(self.handle) and bounty.status == 'done'
         ]
         fulfilled_bounties_count = len(fulfilled_bounties)
+        funded_bounties = [
+            bounty for bounty in bounties if bounty.is_funder(self.handle)
+        ]
+        funded_bounties_count = len(funded_bounties)
+        if funded_bounties_count:
+            total_funded_usd = sum([
+                bounty.value_in_usdt if bounty.value_in_usdt else 0
```

Comment:
We should use [Django Aggregation](https://docs.djangoproject.com/en/2.0/topics/db/aggregation/) here.